src/assignmentB/AssignmentB02.java

```diff
@@ -1,5 +1,96 @@
 package assignmentB;
 
-public class AssignmentB02
+import javax.swing.*;
+import javax.swing.table.DefaultTableModel;
+import java.awt.*;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.Locale;
+
+public class AssignmentB02 extends JFrame
 {
+    Calendar cal = new GregorianCalendar();
+    DefaultTableModel model;
+    JLabel label;
+
+    //TODO: add text box to add/view appointments
+
+    AssignmentB02()
```

Comment:
Instead of creating the Calendar in the `AssignmentB02` constructor, we could create a method called, for example, `createCalendar`, and then invoke that from the constructor or from another method when building the full window.